VS2012.TestAdapter/ChutzpahTestExecutor.cs

```diff
@@ -43,6 +43,7 @@ public void RunTests(IEnumerable<string> sources, IRunContext runContext, IFrame
                         runContext.IsBeingDebugged ? TestLaunchMode.Custom:
                         settings.OpenInBrowser ? TestLaunchMode.FullBrowser:
                         TestLaunchMode.HeadlessBrowser,
+                    BrowserArgs = settings.BrowserArgs,

```

Comment:
I am moving away from adding settings in the VS dialogs. Please add to the chutzpah.json file instead.